azure-pipelines.yml

```diff
@@ -416,14 +412,6 @@ jobs:
       script: >-
         $ZipHost = "mindaromaster.blob.core.windows.net"
 
-        if ($env:BUILD_SOURCEBRANCH -eq "refs/heads/staging") {
```

Comment:
💡☕️ For Archival reasons only please, I believe this is set because we no longer want to keep the staging et. al. anymore?

Reply:
Same reason as previous comment. 
https://github.com/Azure/Bridge-To-Kubernetes/pull/138/files/c7d24b9b9c38cb6925e6eb08ca6aead87b3940e0#r1058975848